cmd/lncli/walletrpc_active.go

```diff
@@ -124,6 +126,45 @@ func getWalletClient(ctx *cli.Context) (walletrpc.WalletKitClient, func()) {
 	return walletrpc.NewWalletKitClient(conn), cleanUp
 }
 
+var estimateOnchainFeeCommand = cli.Command{
+	Name: "estimatefee",
```

Comment:
Since this will return a fee rate (not an absolute fee), I think we should rename the command to `estimatefeerate` (which will also help to differentiate it from the existing top-level `estimatefee` command.